src/richie/apps/search/defaults.py

```diff
@@ -12,6 +12,11 @@
 # Use a lazy to enable easier testing by not defining the value at bootstrap time
 ES_INDICES_PREFIX = lazy(lambda: settings.RICHIE_ES_INDICES_PREFIX)()
 
+# Define which analyzer should be used for each language
+QUERY_ANALYZERS_BY_LANGUAGE = getattr(
+    settings, "RICHIE_QUERY_ANALYZERS_BY_LANGUAGE", {"en": "english", "fr": "french"}
```

Comment:
BY_LANGUAGE may not be necessary. Like the CMS_LANGUAGES setting is in fact a complex dict, you could just call it `QUERY_ANALYZERS`